src/adminSettings.js

```diff
@@ -0,0 +1,10 @@
+import Vue from 'vue'
+import AdminSettings from './views/AdminSettings'
```

Comment:
There are several eslint comments in the src/ files. can you check them?

Reply:
Resolved